helpers/setURLQueryParam.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const URL = require('url').URL;
+const utils = require('handlebars-utils');
+const common = require('./lib/common.js');
+const SafeString = require('handlebars').SafeString;
+
+const factory = () => {
+    return function(string, key, value) {
+        if (string instanceof SafeString) {
+            string = string.toString();
+        }
+        if (!utils.isString(string) || !common.isValidURL(string)){
+            throw new TypeError("Invalid URL passed to setURLQueryParam");
+        }
+        else if (!utils.isString(key)){
+            throw new TypeError("Invalid query parameter key passed to setURLQueryParam");
+        }
+        else if(!utils.isString(value)) {
+            throw new TypeError("Invalid query parameter value passed to setURLQueryParam");
+        } else if (!(key.length > 0) || !(value.length > 0)) {
```

Comment:
Not sure if it will ever come here https://github.com/helpers/handlebars-utils/blob/master/index.js#L490

Reply:
good point, I'll remove